build.sbt

```diff
@@ -382,14 +382,16 @@ lazy val tests =
       // nativeOptimizerReporter := OptimizerReporter.toDirectory(
       //   crossTarget.value),
       sourceGenerators in Compile += Def.task {
-        val dir    = sourceDirectory.value
-        val prefix = dir.getAbsolutePath + "/main/scala/"
+        val dir       = sourceDirectory.value
+        val separator = java.io.File.separator;
+        val prefix = dir.getAbsolutePath + Seq("main", "scala")
+            .mkString(separator, separator, separator)
         val suites = (dir ** "*Suite.scala").get
           .map { f =>
             f.getAbsolutePath
               .replace(prefix, "")
               .replace(".scala", "")
-              .split("/")
+              .split("/|\\\\")
```

Comment:
Thanks, I replaced by Matcher.quoteReplacement(System.getProperty("file.separator") and it works fine